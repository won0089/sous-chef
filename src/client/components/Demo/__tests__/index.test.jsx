import * as React from 'react';
import { shallow } from 'enzyme';
import Demo from 'components/Demo';

describe('<Demo /> tests', () => {
    it('should render', () => {
        const wrapper = shallow(<Demo />);

        expect(wrapper).toMatchSnapshot();
    });
});
