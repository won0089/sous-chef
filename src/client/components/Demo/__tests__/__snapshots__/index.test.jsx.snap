// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Demo /> tests should render 1`] = `
<div
  className="demo"
>
   React + Redux Boilerplate 
</div>
`;
